install/cluster-operator/060-Deployment-strimzi-cluster-operator.yaml

```diff
@@ -26,7 +32,7 @@ spec:
             name: strimzi-cluster-operator
       containers:
         - name: strimzi-cluster-operator
-          image: registry.redhat.io/amq7/amq-streams-rhel8-operator:1.8.0
+          image: registry.redhat.io/amq7/amq-streams-kafka-30-rhel8:2.0.0
```

Comment:
why Kafka, this is the operator image, isn't it?

Reply:
That was a typo, now fixed!